lib/registration.pm

```diff
@@ -120,11 +123,11 @@ sub fill_in_registration_data {
                 }
             }
             send_key 'alt-n';    # next
-            while (check_screen('import-untrusted-gpg-key', 20)) {
+            while (check_screen('import-untrusted-gpg-key')) {
                 record_soft_failure 'untrusted gpg key';
                 send_key 'alt-t', 2;
             }
-            sleep 20;            # scc registration need some time
+            sleep 30;            # scc registration need some time
```

Comment:
put list of abbreviation in comment and removed sleep.
used check_screen rather than assert_screen, because after addons/modules registration, it will assert same successful tag.
and no surprises found :(

verified tests:
http://147.2.207.208/tests/1482
http://147.2.207.208/tests/1483
http://147.2.207.208/tests/1485 

also how about previous mr for needles?
https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/152
I'd like to submit needles of modules after this merged.